### ISSUE TITLE
`useValidAriaProps`: aria-atomic incorrectly flagged as invalid ARIA attribute

### ISSUE THREAD

**Issue by tony19** [biomejs/biome#1342]
### Environment information

 * Biome v1.4.1
 * macOS Sonoma
 * Node v18.18.2

### Description

Even though the [linter supports `aria-atomic`](https://github.com/biomejs/biome/blob/f243c7b/crates/biome_aria_metadata/build.rs#L14), `biome` CLI (and the VS Code Extension) is not recognizing `aria-atomic` as an ARIA attribute.

Sample markup:

```html
<div aria-atomic>Test</div>
<div aria-atomic="true">Test</div>
```

<img width="911" alt="Screenshot 2023-12-25 at 2 40 44 PM" src="https://github.com/biomejs/biome/assets/26580/aae857d5-00af-4843-b5d4-14d99c678692">
<img width="565" alt="Screenshot 2023-12-25 at 3 06 07 PM" src="https://github.com/biomejs/biome/assets/26580/893fea8b-7d7d-4e2d-82b8-f21fc1f57679">

For some reason, the [playground does not exhibit this problem](https://biomejs.dev/playground/?quoteStyle=single&semicolons=as-needed&code=PAA%2BAAoAPABkAGkAdgAgAGEAcgBpAGEALQBhAHQAbwBtAGkAYwA%2BAFQAZQBzAHQAPAAvAGQAaQB2AD4ACgA8AGQAaQB2ACAAYQByAGkAYQAtAGEAdABvAG0AaQBjAD0AIgB0AHIAdQBlACIAPgBUAGUAcwB0ADwALwBkAGkAdgA%2BAAoAPAAvAD4A).

### Rule name

`useValidAriaProps`

### Playground link

https://github.com/tony19-sandbox/biome-aria-atomic-bug

### Expected result

`aria-atomic` should not be flagged as an invalid ARIA attribute

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
Duplicate of https://github.com/biomejs/biome/issues/1167

You can use our latest nightly or wait for the new release.
